AcademicManagerV1/src/com/academic/db/dao/TeacherDao.java

```diff
@@ -0,0 +1,13 @@
+package com.academic.db.dao;
+
+import java.util.List;
+
+import com.academic.model.Person;
+
+public interface TeacherDao extends Dao<Person> {
+	
+	public List<Person> getTeachers();
+	
+	public Person getTeacher(int id);

```

Comment:
This here points to a logical error, as we discussed. However, I will allow it for now.